packages/expo-image-manipulator/android/src/main/java/expo/modules/imagemanipulator/ImageManipulatorModule.kt

```diff
@@ -0,0 +1,89 @@
+package expo.modules.imagemanipulator
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.net.Uri
+import android.os.Bundle
+import android.util.Base64
+import expo.modules.imagemanipulator.arguments.Actions
+import expo.modules.imagemanipulator.arguments.SaveOptions
+import expo.modules.interfaces.imageloader.ImageLoaderInterface
+import expo.modules.interfaces.imageloader.ImageLoaderInterface.ResultListener
+import org.unimodules.core.ExportedModule
+import org.unimodules.core.ModuleRegistry
+import org.unimodules.core.ModuleRegistryDelegate
+import org.unimodules.core.Promise
+import org.unimodules.core.arguments.ReadableArguments
+import org.unimodules.core.interfaces.ExpoMethod
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.FileOutputStream
+import java.io.IOException
+import java.util.*
+
+private const val TAG = "ExpoImageManipulator"
+private const val ERROR_TAG = "E_IMAGE_MANIPULATOR"
+
+class ImageManipulatorModule(
+  context: Context,
+  private val moduleRegistryDelegate: ModuleRegistryDelegate = ModuleRegistryDelegate(),
+) : ExportedModule(context) {
+  private val mImageLoader: ImageLoaderInterface by moduleRegistry()
+
+  private inline fun <reified T> moduleRegistry() = moduleRegistryDelegate.getFromModuleRegistry<T>()
```

Comment:
It's not connected with that particle task, but maybe if we want to use the `moduleRegistryDelegate` in every kotlin module, it will be nice to move that additional logic to the base class (or created a new class `ExportedKotlinModule`)?

cc @tsapeta